includes/tracking/class-pixel.php

```diff
@@ -148,6 +148,11 @@ public static function get_pixel_url( $post_id ) {
 	 * @return void
 	 */
 	public static function track_seen( $newsletter_id, $tracking_id, $email_address ) {
+		// Don't track for logged-in editor or admin users.
+		if ( current_user_can( 'edit_others_posts' ) ) {
+			return;
+		}
+
```

Comment:
This is called on `::process_logs()`, which happens asynchronously and does not have a session.

Reply:
Ah, you're right—I guess we would need to move this to where the logfile is being written to?

Reply:
That would be pretty complex. The purpose of the separate PHP file for the pixel is never to load WP on a pixel hit. Although the cookie exists in the browser, there's nothing to validate it.

Reply:
One way of doing it is to create a new secret token to be stored as a cookie on users that the pixel should skip.

The pixel php can just match that single secret on a cookie and bail.

I'm not a fan, but it works.

Reply:
We can skip this for now—I doubt the extra hits from admins and editors is so great that it's skewing the numbers in a significant way

Reply:
e43d4b2